internal/artifacts/module_arch.go

```diff
@@ -464,12 +465,19 @@ func archiveModuleToResultDir(buildResult string, requestedResultFileName string
 }
 
 // getIgnores - get files and/or subfolders to exclude from the package.
-func getIgnores(module *mta.Module) []string {
+func getIgnores(moduleLoc dir.IModule, module *mta.Module, moduleResultPath string) []string {
 	var ignoreList []string
 	// ignore defined in build params is declared
 	if module.BuildParams != nil && module.BuildParams[ignore] != nil {
 		ignoreList = convert(module.BuildParams[ignore].([]interface{}))
 	}
+	// we add target folder to the list of ignores to avoid it's packaging
```

Comment:
Please add to this comment that the ignored folder is the root where all the build results are created, even if we are building more than one module

Reply:
added